### DISCUSSION TITLE
切换了翻译插件，使用语音转换时报TTS网络连接错误

### DISCUSSION THREAD

**Discussion by zhujianwei-star** [YiiGuxing/TranslationPlugin#2724] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/58215451/206383406-b2e9511d-3efb-4f06-a8fe-cd4fd9bda3ad.png)
![image](https://user-images.githubusercontent.com/58215451/206383528-fa57b9bd-b173-4f25-b8f6-6479c6df9167.png)

**Comment by YiiGuxing** [comment, marked as answer]
请另阅 #2315

**Reply by zhujianwei-star** [reply]
好的，感谢
